src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -643,16 +643,12 @@ public String getQuestionTypeChoiceOption() {
         for(FeedbackResponseAttributes response : responses){
             boolean validAnswer = false;
             FeedbackContributionResponseDetails frd = (FeedbackContributionResponseDetails) response.getResponseDetails();
-            for(int i=200; i>=0; i-=10){
-                if(frd.getAnswer() == i){
-                    validAnswer = true;
-                    break;
-                }
+            if(frd.getAnswer() <= 200 && frd.getAnswer() >= 0 && frd.getAnswer() % 10 == 0) {
```

Comment:
The meaning of this condition may not be apparent straight away. Use 'introduce explaining variable' refactoring?